config/initializers/devise.rb

```diff
@@ -6,7 +6,6 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-   config.secret_key = '34486348de8caa31dbf426f515084b3c4b33c15954ce1b0eb8f664f58eb68453ba7dd2603aa0a6a54ee816adcb5a72adca4564df5d772a7ae45ab1efbf814c9c'
```

Comment:
I recommend not mixing these changes together in the same PR. Updating loofah is a no-brainer merge and deploy. Changing out how a secret is handled is a separate issue for review and deploy consideration.